src/features/sponsor/components/FAQs.tsx

```diff
@@ -65,12 +65,17 @@ export function FAQs() {
         {faqs.map((faq) => (
           <AccordionItem
             key={faq.question}
+            my="1rem"
             bg="white"
             border="0"
             rounded="0.25rem"
           >
             <h2>
-              <AccordionButton my="1rem" py="0.8rem" rounded="0.25rem">
+              <AccordionButton
```

Comment:
need a transition while opening too; currently transition shows up only when closing the accordion

Reply:
The opening transition is not working at all, I tried it for long.
I also tried pasting an example component code from Chakraui's accordion, but the issue persisted there too.

Reply:
try passing props

Reply:
tried all possibilities.

Reply:
Just saw that it's an issue in Chakra UI itself:
https://github.com/chakra-ui/chakra-ui/issues/8448